media-fonts/fantasque-sans-mono/fantasque-sans-mono-1.7.2.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit font
+
+DESCRIPTION="Functional programming font designed with some handwriting-like fuzziness"
+HOMEPAGE="https://github.com/belluzj/fantasque-sans"
+SRC_URI="https://github.com/belluzj/fantasque-sans/releases/download/v${PV}/FantasqueSansMono-Normal.tar.gz -> ${P}.tar.gz"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
+
+S="${WORKDIR}"
+FONT_S="${S}/OTF"
+FONT_SUFFIX="otf"
+DOCS="README.md"
```

Comment:
This should be installed without defining it.

Reply:
Just copied the existing ebuild, but I've tried removing the lines and build fails without S:

```
* ERROR: media-fonts/fantasque-sans-mono-1.7.2::clusterfuck failed (prepare phase):
 *   The source directory '/var/tmp/portage/media-fonts/fantasque-sans-mono-1.7.2/work/fantasque-sans-mono-1.7.2' doesn't exist
 * 
 * Call stack:
 *            ebuild.sh, line  773:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1071:  Called __dyn_prepare
 *   phase-functions.sh, line  384:  Called die
 * The specific snippet of code:
 *   		die "The source directory '${S}' doesn't exist"
```

With S set but FONT_S, FONT_SUFFIX and DOCS unset it does not install anything other than fonts.scale, encodings.dir, fonts.dir.

Reply:
Well, I only meant the DOCS part...